.github/workflows/stale.yml

```diff
@@ -31,3 +35,16 @@ jobs:
         days-before-issue-close: 30
         close-pr-message: 'This PR is stale and had no activity for too long - it will now be closed.'
         close-issue-message: 'This issue is stale and had no activity for too long - it will now be closed.'
+
+  # Handle exempted issues
+    - uses: actions/stale@v5
+      with:
+
+        days-before-stale: 180        
+        days-before-issue-stale: 180
```

Comment:
where is the days-before-pr-stale? is it intentionally missing here?

Reply:
PRs don't have the labels to make them exempted so they will always be processed by thee 1st rule